### ISSUE TITLE
model for single read  

### ISSUE THREAD

**Issue by Yang990-sys** [sihaohuanguc/Nanopore_psU#3]
Dear author,

I have read your paper and used your package to find psU sites. But I have a question about Single read Ψ analysis. I didn't find any scripts and models about single read analysis in this package. Do you have a optimization plan about this part ,or this part won't be released?

**Comment by DelphIONe** [comment]
Yes! I have the same question... where is the information about single read ?

Thanks for your answer!
